Samples/ResourceManagement/cpp-console-unpackaged/console_unpackaged_app.cpp

```diff
@@ -59,6 +60,13 @@ int wmain(int argc, wchar_t* argv[])
         return 1;
     }
 
+    // Despite the name of this API, it will return FALSE if the user is an admin but the context isn't elevated.
+    if (IsUserAnAdmin())
```

Comment:
What's this? Where's the implementation?

What does "is an admin" mean? Can only such admins be elevated?
See FailFastIfElevated for the actual check done by DynDep that you're trying to work around